tests/snapshot.test.ts

```diff
@@ -42,3 +42,66 @@ it('should not change snapshot with assigning same object', async () => {
   const snap2 = snapshot(state)
   expect(snap1).toBe(snap2)
 })
+
+it('should cache object getters', () => {
```

Comment:
This test passed w/o any code changes, just added it to document/cover the current behavior.